src/main/java/uk/gov/cslearning/record/domain/Booking.java

```diff
@@ -0,0 +1,32 @@
+package uk.gov.cslearning.record.domain;
+
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import javax.persistence.*;
+import java.time.LocalDateTime;
+
+@Entity
+@Data
+@NoArgsConstructor
+public class Booking {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long id;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name="learnerId")
+    private Learner learner;
+
+    @ManyToOne(cascade = CascadeType.PERSIST)
+    @JoinColumn(name="eventId")
+    private Event event;
+
+    private String paymentDetails;
+
+    @Column(nullable = false)
+    private String status;
+
+    private LocalDateTime bookingTime;
```

Comment:
Consistency - As Peter said, consider using Instant, please? If there is a reason why we shouldn't use Instant please give us a shout.